start.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is the wrong header. Use the same one at the top of [start.py from topology_nodebase](https://github.com/clusterdock/topology_nodebase/blob/master/start.py).

Reply:
Done.